UI/EmuScreen.cpp

```diff
@@ -987,6 +987,18 @@ void EmuScreen::update() {
 	resumeButton_->SetVisibility(coreState == CoreState::CORE_RUNTIME_ERROR && Memory::MemFault_MayBeResumable() ? V_VISIBLE : V_GONE);
 	resetButton_->SetVisibility(coreState == CoreState::CORE_RUNTIME_ERROR ? V_VISIBLE : V_GONE);
 
+	if (chatButton_ && chatMenu_) {
+		if (chatMenu_->GetVisibility() != V_GONE) {
+			chatMessages_ = GetChatMessageCount();
+			newChatMessages_ = 0;
+		} else {
+			int diff = GetChatMessageCount() - chatMessages_;
+			chatMessages_ += diff;
```

Comment:
`chatMessages_` should not be updated or it will display 0 the next frame.

Reply:
Not sure what I was thinking, I added that afterward but I really can't remember why and I somehow thought it was needed as I was typing it.  Must be my insanity finally taking over.

-[Unknown]